### PR TITLE
Documentation for the XLG Debugger

### PR DESCRIPTION
**Pull request by almiliMSFT**

### CODE REVIEW

**Comment by almiliMSFT** [comment]
Link to live docs

[https://github.com/almiliMSFT/BuildXL/blob/pr/xlg-docs/Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XlgDebugger/Documentation/INDEX.md](https://github.com/almiliMSFT/BuildXL/blob/pr/xlg-docs/Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XlgDebugger/Documentation/INDEX.md)

File: Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XlgDebugger/Documentation/INDEX.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# BuildXL eXecution LoG Debugger
+
+XLG Debugger is a Visual Studio Code debugger extension for viewing/exploring/diagnosing/debugging BuildXL log files.
```
Comment (Kr1sso) [inline Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XlgDebugger/Documentation/INDEX.md]: Nit: binary log files

Comment (almiliMSFT) [inline Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XlgDebugger/Documentation/INDEX.md]:
```suggestion
XLG Debugger is a Visual Studio Code debugger extension for viewing/exploring/diagnosing/debugging BuildXL binary log files.
```

Comment (almiliMSFT) [reply Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XlgDebugger/Documentation/INDEX.md]: done

**Pull request merged**